Cargo.toml

```
@@ -11,21 +11,25 @@ license = "MIT"
 keywords = ["sprite", "svg", "cartography", "vector-tiles", "maplibre"]
 categories = ["command-line-utilities", "encoding", "filesystem", "graphics"]
 
+[features]
+default = ["cli"]
+cli = ["clap", "exitcode", "rayon"]
```

```suggestion
cli = ["dep:clap", "dep:exitcode", "dep:rayon"]
```

Spreet's minimum Rust version is 1.61, which means we can use [Cargo's newish `dep:` syntax](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies) to disable the implicit features otherwise created here